net/net.go

```diff
@@ -764,7 +785,7 @@ func (n *net) putRecord(ctx context.Context, id thread.ID, lid peer.ID, rec core
 		return nil
 	}
 	// Get or create a log for the new rec
```

Comment:
Not a PR change here, but this should probably read "Get the log" (nothing about create), or just be removed

Reply:
Done